protocol/0043-MKTL-market_lifecycle.md

```diff
@@ -316,15 +316,15 @@ The market state is `active`.
 When this is approved and enacted the market state is `closed`.
 Parties that had open positions see settlement cash-flows happen to settle positions.
 Margin account balances are transferred to the general account.
-After `market.liquidity.successorLaunchWindowLength` has passed since market settlement, any insurance pool balance is [redistributed](./0015-INSR-market_insurance_pool_collateral.md) equally between the global insurance pool and the insurance pools of the remaining active markets using the same settlement asset.
+After `market.liquidity.successorLaunchWindowLength` has passed since market settlement, any insurance pool balance is [redistributed](./0015-INSR-market_insurance_pool_collateral.md) into the global insurance pool using the same settlement asset.
```

Comment:
```suggestion
After `market.liquidity.successorLaunchWindowLength` has passed since market settlement, any [insurance pool](./0015-INSR-market_insurance_pool_collateral.md) balance is transferred into the global insurance pool using the same settlement asset.
```